.travis.yml

```diff
@@ -8,8 +8,11 @@ script:
 after_success:
   - '[ -d coverage ] && bundle exec codeclimate-test-reporter'
 rvm:
+  - 2.0
+  - 2.1
   - 2.2
   - 2.3.1
+  - rbx-3
```

Comment:
How come this is part of this PR? :/

Reply:
@solnic looks like rbx-3 was _removed_ as part of https://github.com/dry-rb/dry-configurable/pull/49. Since I'm reverting the _merge commits_ for the recent PRs, it gets added back here.

It's kind of a bother this was changed within the PR, but rather than fiddling around and reverting every individual commit within the PRs (except this one), I might just go back and fix up the .travis.yml afterwards, in a separate commit (it needs new rubies anyway).

Reply:
d'ooh of course, thanks!